dts/arm/st/wl/stm32wl.dtsi

```diff
@@ -292,7 +292,7 @@
 			label = "SUBGHZSPI";
 			use-subghzspi-nss;
 
-			radio@0 {
+			lora@0 {
```

Comment:
This doesn't feel right, this radio can be used for other purposes than lora ((G)FSK, (G)MSK, and BPSK.)

Reply:
This is a Lora device that can work in multiple modulation modes you mentioned. In all the cases, it is sold as a "LoRa module" only, so the node name matches.

Reply:
I disagree, this node is common to the whole stm32wl seriers.
Looking for instance to the STM32WLE ref manual:
Headline: 
```
STM32WLEx advanced Arm ® -based 32-bit MCUs with sub-GHz radio solution
```
Description: 
```
STM32WLEx microcontrollers with integrated sub-GHZ radio operating in the
150 - 960 MHz ISM band, belong to a family of microcontrollers with different memory sizes,
packages and peripherals.
```
Radio subystem description:
```
4.5 Sub-GHz radio modems
The following modems are provided enabling the respective modulation and associated
framing.
• LoRa with the LoRa framing
• FSK and MSK with a generic framing
• BPSK with a BPSK framing
```

Lora is clearly a one of multiple modems, not the only one

Reply:
Ah, sorry. I overlooked the STM32WL part. This is clearly different from the LoRa modules available in the market as it supports multiple modulations in a single chip. But shouldn't this be documented in devicetree spec?